exercises/ml-basics/ex_rnw/ex_ml_proj.Rnw

```diff
@@ -0,0 +1,56 @@
+Imagine you work at a second-hand car dealer and are tasked with finding 
+for-sale vehicles your company can acquire at a reasonable price. You decide to 
+address this challenge in a data-driven manner and develop a model that predicts 
+adequate market prices (in EUR) from vehicles’ properties.
+
+\begin{enumerate}[a)]
+
+  \item Characterize the task at hand: supervised or unsupervised? Regression or 
+  classification? Learning to explain or learning to predict? Justify your 
+  answers.
+  \item How would you set up your data? Name potential features along with their 
+  respective data type and state which is the target variable.
+  \item Assume now that you have data on vehicles’ age, mileage, and price. 
+  Explicitly define the feature space $\Xspace$ and target space $\Yspace$.
+  \item You choose to use a linear model (LM) for this task.
+  For this, you assume the targets to be conditionally independent given the 
+  features, i.e., $\yi|\xi \ind \yi[j]|\xi[j]$ for all $i,j \in 
+  \{1, 2, \dots, n\}, i \neq j$, with sample size $n$.
+  The LM models the target as a linear function of the features 
+  with Gaussian error term: $\ydat = \Xmat \thetab + \epsilon$, \\ 
+  $\epsilon \sim N(\bm{0}, \mathit{diag}(\sigma^2)), ~~ \sigma > 0$.
+  % Furthermore, you have reason to believe that the effect of mileage might be 
+  % non-linear, so you decide to include this quantity logarithmically (using the 
+  % natural logarithm).
+  State the hypothesis space for the corresponding model class.
+  Which parameters need to be learned?
```

Comment:
Die Frage würde ich in die nächste Teilaufgabe schieben, dann passt es besser zum Parameter space.